snow/consensus/snowball/consensus_test.go

```diff
@@ -16,16 +16,18 @@ var (
 	_ Consensus = (*Byzantine)(nil)
 )
 
+func NewByzantine(_ Parameters, choice ids.ID) Consensus {
```

Comment:
Can we get rid of the unused parameter here? Looks like we only had it previous for interface signature